tests/test_uroot.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+# Clone repo
+git clone --branch "${VERIFICATION_TEST_UROOT_VERSION}" --depth 1 https://github.com/u-root/u-root.git
+cd u-root
+
+# Make
+go build
+./u-root \
+	-defaultsh gosh \
+	-o initramfs.cpio \
+	boot coreboot-app \
+	./cmds/core/* \
+	./cmds/boot/*
```

Comment:
Shouldn't this already be part of the `boot` preset?

Reply:
Yeah, when I look at `u-root`'s `README` I see that `u-root core boot` should be enough.